docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+# Getting Started
+
+This document introduces the basic steps needed to install all components of cloudflow and deploy and run a simple cloudflow application on the GKE cluster.
+
+We will discuss the following steps in sequence:
+
+1. Download and install cloudflow CLI
```

Comment:
I suggest we start with building the app and separate deployment steps eg. downloading the cli and running Cloudflow on GKE. This is a bit of a long list, let's split it to high level steps here.